spec/features/cases/offender_sar/case_search_spec.rb

```diff
@@ -100,9 +100,9 @@
     let(:searcher) { responder }
     let(:search_term) { "Common text" }
 
-    scenario 'finds offender_sar_case but not FOI' do
-      expect(cases_search_page.search_results_count.text).to eq "1 case found"
-      expect(cases_search_page.case_list.count).to eq 1
+    scenario 'finds offender_sar_case and FOI' do
```

Comment:
How does this find an FOI case when one is not defined in the spec?

Reply:
Ah I see its called `kase` - might be a bit more readable to change it from `kase` to `foi_kase` in the `given()` on line `7` so that it's clearer that its an FOI we are creating and then searching for later